app/components/album-art.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+import layout from '../templates/components/album-art';
+
+export default Ember.Component.extend({
+  layout: layout,
+  classNames: ["album-art"],
+  classNameBinding: ["loading"],
+  getSize: function() {
+    this.set("size", $("img", this.element).width());
+  }.on("didInsertElement"),
+
+  getImage: function() {
+    var album = this.get("album");
+    try {
+      this.set("imageSource", album.images[0]);
+    } catch (_) {
+      $.getJSON(`http://itunes.apple.com/search?term=${encodeURIComponent(album.name)}&limit=1&entity=album&callback=?`)
```

Comment:
Do you have some example so I can test fetching it? I tried some queries, but the album picture url was always returned altogether with album object